src/utils/api.ts

```diff
@@ -4,6 +4,7 @@ import { Database as HypercertsDatabase } from "./hypercerts-database-types";
 import { CONSTANTS, parseClaimOrFractionId } from "@hypercerts-org/sdk";
 import { getFractionsById, getOrders } from "./graphl";
 import { cacheExchange, Client, fetchExchange } from "@urql/core";
+import { deprecate } from "util";
```

Comment:
unused import? Type from @deprecated?

Reply:
Good catch. Got imported automatically when trying to mark a method as deprecated in the js docs.